### PR TITLE
[homework#5]-HTML-CSS by Igor Ignatenko

### PR DESCRIPTION
**Pull request by Memoriarty1**
image
image
image

### CODE REVIEW

**Comment by Memoriarty1** [comment]
image
image

File: homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css [hunk 1]
```diff
@@ -0,0 +1,246 @@
+*, *:after, *:before {
+    box-sizing: border-box;
+    font-size: 1rem;
+    color:white
```
Comment (esvyridov) [inline homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css:4]: let's use `box-sizing` only in `*` selector

File: homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    width: 100%;
+    margin: 0 auto
+}
+
+/*BUTTON*/
+
+.wave-btn__container {
+    padding: 10px;
+    margin-bottom: 10px;
+    overflow: hidden;
```
Comment (esvyridov) [inline homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css:32]: can be removed

File: homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+    align-items: center;
+    border-radius:  10px;
+    transition:  transform 1.5s ease 0s;
+    text-decoration: none;
+    position: relative;
+    overflow: hidden;
+}
+
+.wave-btn:hover {
+    transition: top 0.8s ease 0.2s;
```
Comment (esvyridov) [inline homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css:52]: do we need this? I don't see any effect on layout

Comment (esvyridov) [reply homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css:52]: oh, probably this should be on `..wave-btn__waves`. Anyway, you should try to `transition` `transform` instead

File: homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+.wave-btn:hover .wave-btn__waves {
+    top: -50px
+}
+
+.wave-btn__text {
+    font-family: Arial, Helvetica, sans-serif;
+    color: white;
+    text-transform: uppercase;
+    letter-spacing: 6px;
+    position: relative;
```
Comment (esvyridov) [inline homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css:65]: can be removed

File: homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+    padding: 10px;
+    height: 100%;
+    width: 100%;
+    opacity: 0;
+    background-color: rgba(0, 0, 0, 0.5);
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    text-align: center;
```
Comment (esvyridov) [inline homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css:161]: can be removed

File: homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+    transform: translateY(+200px);
+    transition: transform 0.6s ease, opacity 0.5s ease;
+    opacity: 0;
+}
+
+.gallery__button:active {
+    color: lightpink;
+}
+
+.gallery__heading {
```
Comment (esvyridov) [inline homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css:181]: it seems this styles can be combined with https://github.com/qaprosoft/devlaba01/pull/59/files#diff-937398f93d1d8d0af4f8db73a472da514a7abc354d88a59167a139019d85c489R140

File: homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+}
+.gallery__text {
+    opacity: 0;
+    font-size: 0.9rem;
+    transform: translateY(+200px);
+    transition: transform 0.6s ease, opacity 0.3s ease;
+}
+
+.gallery__content:hover {
+    opacity: 1;
```
Comment (esvyridov) [inline homeworks/igor-ignatenko/05-html-css/fake portfolio/style.css:195]: I would change `background-color: rgba(0, 0, 0, 0.5);` to `background-color:  #000` in `.gallery__content` and animate opacity on hover to `.5`

File: homeworks/igor-ignatenko/05-html-css/fake portfolio/index.html [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+                <div class="gallery__item"
+                    style="background-image: url(https://72tv.ru/uploads/posts/2018-08/1534927590_0165-bmpzj3obhi4.jpg)">
+                    <div class="gallery__content">
+                        <h2 class="gallery__heading">Project name</h2>
+                        <p class="gallery__text">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+                        <a class="gallery__button">Read</a>
+                    </div>
+                </div>
+                <div class="gallery__item"
+                    style="background-image: url(https://www.ejin.ru/wp-content/uploads/2017/09/1-887.jpg)">
```
Comment (esvyridov) [inline homeworks/igor-ignatenko/05-html-css/fake portfolio/index.html:42]: the picture isn't optimized

File: homeworks/igor-ignatenko/05-html-css/fake portfolio/index.html [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+                <div class="gallery__item "
+                    style="background-image: url(https://natworld.info/wp-content/uploads/2018/01/-%D0%BF%D1%80%D0%B8%D1%80%D0%BE%D0%B4%D1%8B-e1534007720297-900x500.jpg)">
+                    <div class="gallery__content">
+                        <h2 class="gallery__heading">Project name</h2>
+                        <p class="gallery__text">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+                        <a class="gallery__button">Read</a>
+                    </div>
+                </div>
+                <div class="gallery__item _big-right"
+                    style="background-image: url(https://www.ejin.ru/wp-content/uploads/2019/05/zakat-zimoj-na-gore.jpg)">
```
Comment (esvyridov) [inline homeworks/igor-ignatenko/05-html-css/fake portfolio/index.html:58]: please optimize the picture

File: homeworks/igor-ignatenko/05-html-css/fake portfolio/index.html [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+                <div class="gallery__item"
+                    style="background-image: url(https://zefirka.net/wp-content/uploads/2018/12/pejzazhi-i-priroda-kanady-na-snimkax-robin-lorenson-1.jpg)">
+                    <div class="gallery__content">
+                        <h2 class="gallery__heading">Project name</h2>
+                        <p class="gallery__text">Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+                        <a class="gallery__button">Read</a>
+                    </div>
+                </div>
+                <div class="gallery__item _big-left"
+                    style="background-image: url(https://iso.500px.com/wp-content/uploads/2016/03/stock-photo-142984111.jpg)">
```
Comment (esvyridov) [inline homeworks/igor-ignatenko/05-html-css/fake portfolio/index.html:76]: please optimize the picture

**Review by esvyridov — APPROVED** [review]

**Pull request merged**